app/views/search/_filter.html.erb

```diff
@@ -1,19 +1,28 @@
 <% return if values.empty? %>
 
 <div class="category">
-  <h3><%= nice_labels[category] || category %></h3>
-  <ul class="category-terms list-unbulleted">
-  <% values.each do |term| %>
-    <li class="term">
-      <a href="<%= results_path(add_filter(@enhanced_query, category, term['key'])) %>" class="<%= "applied" if @enhanced_query[category.to_sym] == term['key'] %>">
-        <span class="name"><%= term['key'] %></span>
-        <span class="count"><%= term['docCount'] %> <span class="sr">records</span></span>
-      </a>
-    </li>
-  <% end %>
-  </ul>
-  <% if @enhanced_query[category.to_sym].present? %>
-    <div><%= link_to "Show all #{nice_labels[category]&.downcase || category}", results_path(remove_filter(@enhanced_query, category)) %>
-    </div>
-  <% end %>
+  <button class="filter-label <%= 'expanded' if @enhanced_query[category.to_sym].present? %>"
+          onClick="toggleFilter(this)"><%= nice_labels[category] || category %></button>
+  <div class="filter-options">
+    <ul class="category-terms list-unbulleted">
+    <% values.each do |term| %>
+      <li class="term">
```

Comment:
I'm not entirely sure why, but the scroll overlaps the content initially in Safari. If I select an element (via tab for example) it reflows the display and looks correct.


Expanded but with nothing selected the scrollbar overlaps the content
<img width="279" alt="Screenshot 2024-02-05 at 2 35 09 PM" src="https://github.com/MITLibraries/timdex-ui/assets/1386650/2ddedb44-72c2-4bc0-9762-db6bbabff8f6">

Tabbed into an element, the overlap is fixed
<img width="267" alt="Screenshot 2024-02-05 at 3 42 42 PM" src="https://github.com/MITLibraries/timdex-ui/assets/1386650/234e3885-2b31-43e6-9dea-08e2fd96b76a">